### PR TITLE
Add Quarkus to Supported migration paths table

### PR DESCRIPTION
**Pull request by RichardHoch**
Resolves "Docs issue -- migration matrix update" in https://docs.google.com/spreadsheets/d/1av_c1ZrBS6IhClXJE3sgQnd_7eXa2rxs_0SIv4lyUqM/edit?ts=5fcdfdf3#gid=0

Preview: http://file.emea.redhat.com/rhoch/10122020/Matrix/html-single/#migration_paths_getting-started-guide

### CODE REVIEW

**Comment by RichardHoch** [comment]
@apinnick: please review.

File: docs/topics_5/migration-paths.adoc [hunk 1]
```diff
@@ -20,6 +20,7 @@ The following table describes the most common supported migration paths.
 |OpenJDK 8{nbsp}&{nbsp}11
```
Comment (apinnick) [inline docs/topics_5/migration-paths.adoc:20]: OpenShift Container Platform is not rendering well in the header. In the paragraph above the table (MTA provides a comprehensive set of rules to assess the suitability of your applications for containerization and deployment on Red Hat OpenShift Container Platform. You can run an MTA analysis to assess your applications' suitability for migration to multiple target platforms.), change 'Red Hat OpenShift Container Platform' to 'Red Hat OpenShift Container Platform (RHOCP)' and use RHOCP in the table header.

File: docs/topics_5/migration-paths.adoc [hunk 2]
```diff
@@ -84,14 +93,16 @@ The following table describes the most common supported migration paths.
...
 |-
 |-
 |{icon-check}
 |{icon-check}
-|{icon-check}
 |-
+|{icon-check}
+|{icon-check}
 
 |Any Java application
```
Comment (apinnick) [inline docs/topics_5/migration-paths.adoc:107]: I'm not crazy about the word "Any". I think this could be changed to "Java application". Your call.

**Comment by RichardHoch** [comment]
@apinnick: please review again.

**Pull request merged**